### DISCUSSION TITLE
Running API on HTTPS. "Mixed content" error

### DISCUSSION THREAD

**Discussion by Sl8my** [saleor/saleor#10515] [Deployment]
I have in my server the saleor core running. I basically followed this guide [guide] https://github.com/neerajgupta2407/saleor_course_resources/blob/master/setup_saleor_core.txt).
I then built the Dashboard and uploaded  only the build folder in my hosting web. After some debug I was able to see the login page in
myurl.com/dashboard. Now the problem is that the /dashboard page is https, the API URL not. 

![image](https://user-images.githubusercontent.com/52766235/187238287-08bb183c-4576-4125-8330-d0f6f9c40629.png)

Im in production so I would like to have the API running in HTTPS.
Thank you very much.

**Comment by ranjithmahadev** [comment]
@Sl8my Did you find a solution for this?
